java/ql/src/Security/CWE/CWE-200/SpringBootActuators.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Exposed Spring Boot actuators
+ * @description Exposing Spring Boot actuators may lead to internal application's information leak
```

Comment:
Perhaps, to simplify and improve readability? There a few stacked modifiers, so suggesting the following (feel free to rewrite if I've misunderstood(
```suggestion
 * @description Exposing Spring Boot actuators may lead to information leak from the internal application,
```